src/components/CardEventList/styles.scss

```diff
@@ -22,4 +22,17 @@
         overflow-y: scroll;
         margin-bottom: 0;
     }
+}
+
+.results-btn {
+    width: 272px;
+    // height: 44px;
+    // margin-top: 32px;
```

Comment:
Delete this line

Reply:
Hi again, I delate the comments line.

I improve the button function too.
THe problem was that the content of the component was inside the conditional, so it "dessapear" when it's loading and rendering again.
To fix it the solution was write it outside the conditional and include a class hidden when it's loading